pydantic_ai_slim/pydantic_ai/direct.py

```diff
@@ -0,0 +1,217 @@
+
+
+    async def main():
+        model_response = await model_request(
+            'anthropic:claude-3-5-haiku-latest',
+            [ModelRequest.user_text_prompt('What is the capital of France?')]  # (1)!
+        )
+        print(model_response)
+        '''
+        ModelResponse(
+            parts=[TextPart(content='Paris', part_kind='text')],
+            usage=Usage(
+                requests=1,
+                request_tokens=56,
+                response_tokens=1,
+                total_tokens=57,
+                details=None,
+            ),
+            model_name='claude-3-5-haiku-latest',
+            timestamp=datetime.datetime(...),
+            kind='response',
+        )
+        '''
+    ```
+
+    1. See [`ModelRequest.user_text_prompt`][pydantic_ai.messages.ModelRequest.user_text_prompt] for details.
+
+    Then
+
+    Args:
+        model: The model to make a request to. We allow `str` here since the actual list of allowed models changes frequently.
+        messages: Messages to send to the model
+        model_settings: optional model settings
+        model_request_parameters: optional model request parameters
+        instrument: Whether to instrument the request with OpenTelemetry/Logfire, if `None` the value from
+            [`logfire.instrument_pydantic_ai`][logfire.Logfire.instrument_pydantic_ai] is used.
+
+    Returns:
+        The model response and token usage associated with the request.
+    """
+    model_instance = _prepare_model(model, instrument)
+    return await model_instance.request(
+        messages,
+        model_settings,
+        model_instance.customize_request_parameters(model_request_parameters or models.ModelRequestParameters()),
+    )
+
+
+def model_request_sync(
+    model: models.Model | models.KnownModelName | str,
+    messages: list[messages.ModelMessage],
+    *,
+    model_settings: settings.ModelSettings | None = None,
+    model_request_parameters: models.ModelRequestParameters | None = None,
+    instrument: instrumented_models.InstrumentationSettings | bool | None = None,
+) -> messages.ModelResponse:
+    """Make a Synchronous, non-streamed request to a model.
+
+    This is a convenience method that wraps [`model_request`][pydantic_ai.direct.model_request] with
+    `loop.run_until_complete(...)`. You therefore can't use this method inside async code or if there's an active event loop.
+
+    ```py title="model_request_sync_example.py"
+    from pydantic_ai.direct import model_request_sync
+    from pydantic_ai.messages import ModelRequest
+
+    model_response = model_request_sync(
+        'anthropic:claude-3-5-haiku-latest',
+        [ModelRequest.user_text_prompt('What is the capital of France?')]
+    )
+    print(model_response)
+    '''
+    ModelResponse(
+        parts=[TextPart(content='Paris', part_kind='text')],
+        usage=Usage(
+            requests=1, request_tokens=56, response_tokens=1, total_tokens=57, details=None
```

Comment:
```suggestion
            requests=1, request_tokens=56, response_tokens=1, total_tokens=57, details=None,
```

Just so that it expands and makes it easier to diff the methods